docs/design/layout.md

```diff
@@ -369,4 +369,83 @@ So even though a `measureSpec` is an integer, you cannot simply add or subtract
 
 It is also important to remember that measureSpec size values should always be in native pixels. This means that when constructing a measureSpec from cross-platform sizes, you need to convert the sizes using `Context.ToPixels()`. 
 
+# Legacy Notes
 
+## Xamarin.Forms -> MAUI.Controls Layout Differences
+
+### StackLayout
+ 
+There are a few differences between the stack layouts in MAUI (StackLayout, VerticalStackLayout, and HorizontalStackLayout) and the StackLayout in Xamarin.Forms. The first is that the MAUI stack layouts are _very_ simple; they stack their child views in a single direction until all of them have been stacked. They will keep going until the last child has been stacked, even if that takes them beyond the available space in the stacking direction. MAUI stack layouts simply arrange controls in a particular direction; they do not subdivide a space. 
```

Comment:
Out of curiosity, is there a performance difference in rendering StackLayout children in MAUI compared to Xamarin? If so, I think there is value documenting that here or elsewhere.

Reply:
I've added a note about this in the next paragraph.